newapkbuild.1

```diff
@@ -0,0 +1,220 @@
+directory if
+.Ar source_url
+is specified, and write out typical build instructions for that build system.
+If you do not specify the source URL, or you want to override auto-detection,
+you may specify one of the following four flags:
+
+.Bl -tag -width "-a" -offset indent -compact
+
+.It Fl a
+Specifies that the package uses autotools.  The APKBUILD will contain a typical
+invocation of ./configure, make, and make install.
+
+.It Fl C
+Specifies that the package uses CMake.  CMake will be added to the makedepends
+and a typical CMake invocation will be added to the APKBUILD.
+
+.It Fl m
+Specifies that the package uses Meson.  A typical Meson and Ninja invocation
+will be added to the APKBUILD.
+
+.It Fl p
+Specifies that the package uses a Perl Makefile.PL file.  The CPAN template
+will be used.
+
+.It Fl y
+Specifies that the package uses a Python setup.py build system.  Python will be
+added to the makedepends.
+
+.El
+
+If you do not specify any option, and you do not specify the source URL, the
+generated APKBUILD file will not contain any build instructions.
+
+
+.Sh FILES
+
+All files generated will be placed in a
+.Pa pkgname
+directory inside the current working directory, with
+.Pa pkgname
+being created if it does not exist.
+
+.Bl -tag -width "pkgname.post-install" -compact
+
+.It Pa APKBUILD
+.Nm
+will create
+.Pa APKBUILD
+with the details gathered from the invocation of
+.Nm ,
+and introspection of the downloaded package source if
+.Ar source_url
+is provided.
+
+For more information about APKBUILD and its format, see
+.Xr APKBUILD 5 .
+
+.It Pa pkgname.initd
+If
+.Fl c
+is given,
+.Nm
+will create
+.Pa pkgname.initd
+with example data to assist in the creation of an init.d script for a daemon.
+
+.It Pa pkgname.confd
+If
+.Fl c
+is given,
+.Nm
+will create
+.Pa pkgname.confd
+to assist in the creation of a conf.d file for a daemon, used by init.d
+scripts.  conf.d files are used to configure init.d scripts; for more
+information, see
+.Xr openrc 8
+and
+.Xr rc_config 3 .
+
+.It Pa pkgname.pre-install
+If
+.Fl c
+is given,
+.Nm
+will create
+.Pa pkgname.pre-install ,
+the contents of which will be run by
+.Xr apk 8
+before the package is installed.
+
+.It Pa pkgname.post-install
+If
+.Fl c
+is given,
+.Nm
+will create
+.Pa pkgname.post-install ,
+the contents of which will be run by
+.Xr apk 8
+after the package is successfully installed.
+
+For more information about apk install hooks, consult the
+.Xr apk 8
+manual.
+
+.El
+
+
+.Sh EXAMPLES
+
+newapkbuild -n sharutils -d "Utilities for manipulating shell archives" \\
+            -l "GPL-3.0+" -u "https://www.gnu.org/software/sharutils/" -a \\
+            https://ftp.gnu.org/gnu/sharutils/sharutils-4.15.2.tar.xz
+
+
+.Sh SEE ALSO
+
+SPDX license reference (on the Web at <https://spdx.org/licenses/>),
+.Xr abuild 1 ,
+.Xr apk 8 ,
+.Xr APKBUILD 5 .
+
+
+.Sh AUTHORS
+
+.Nm :
+.An Natanael Copa Aq ncopa@alpinelinux.org
+
+Documentation:
+.An A. Wilcox Aq awilfox@adelielinux.org
```

Comment:
Both of these should be `Aq Mt` instead of just `Aq`, sorry.  I will fix this in the next revision.